### DISCUSSION TITLE
Redirect but keep original URL?

### DISCUSSION THREAD

**Discussion by bitttttten** [vercel/vercel#6543] [Help] [answered]
Is it possible to set up a redirect that keeps the original URL? Something like a ProxyPass?

Why I am looking to use it, is that I have a serverless function that returns an image. You can access this image at `/api/image`. Although it would be nice if the URL is something like `/image/example.jpg`

I can setup this redirect:

```json
{
  "redirects": [
    {
      "source": "/image/example.jpg",
      "destination": "/api/image"
    },
  ]
}
```

This works, but it forwards to `/api/image` but it would be nice to keep the original URL. Is it possible?

**Comment by leerob** [comment, marked as answer]
Yes, using [rewrites](https://vercel.com/docs/configuration#project/rewrites)!

```js
{
  "rewrites": [
    {
      "source": "/image/example.jpg",
      "destination": "/api/image"
    },
  ]
 }
 ```
